tests/test_router_bgp.rb

```diff
@@ -311,6 +311,24 @@ def test_routerbgp_default_cluster_id
     bgp.destroy
   end
 
+  def test_routerbgp_set_get_enforce_first_as
+    asnum = 55
```

Comment:
Please test this property under both the default and non-default vrf.  There are examples of this for the other properties.